docs/components/connectors/use-connectors/index.md

```diff
@@ -48,22 +48,54 @@ For further details on how secrets are implemented in Connectors, consult our [C
 Using this in other areas can lead to unexpected results and incidents.
 :::
 
-## Response mapping
+## Variable/response mapping
 
-Most Connectors have a `Response Mapping` section that typically consists of two fields: `Result Variable` and `Result Expression`. These fields are used to export responses from an external Connector call into process variables.
+In most instances, **Connectors** feature a dedicated section known as `Response Mapping`,
```

Comment:
In which instances will they not feature this section?

Reply:
Added